simplq/src/components/common/QueueInfo/QueueInfo.module.scss

```diff
@@ -3,14 +3,12 @@
 .detail {
   display: flex;
   flex-direction: column;
-  padding: 0.5rem 2rem;
-  max-width: 35rem;
+  padding: 2rem 0rem;
 
   .detail-row {
     display: flex;
     align-items: center;
     justify-content: space-between;
-    height: 1.5rem;
```

Comment:
This was causing the text to overlap on smaller screens
![Screenshot 2021-03-27 at 11 13 45 AM](https://user-images.githubusercontent.com/11256376/112712507-16c46200-8ef6-11eb-9c11-6f49f8807305.png)